api/v1alpha/api.pb.go

```diff
@@ -224,7 +224,7 @@ type Image struct {
 	// Base format of the image, required. This indicates the original format
 	// for the image as nowadays all the image formats will be transformed to
 	// ACI.
-	BaseFormat *ImageFormat `protobuf:"bytes,1,opt,name=base_format" json:"base_format,omitempty"`
+	BaseFormat *ImageFormat `protobuf:"bytes,1,opt,name=base_format,json=baseFormat" json:"base_format,omitempty"`
```

Comment:
Where does this change come from?

Does it stay compatible with previous versions? I don't know the default case for the `json=` attribute.